src/components/LoansByCategory/MFIRecommendations/MFIHero.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<div class="tw-w-full">
+		<div class="tw-flex tw-items-center tw-gap-1 tw-px-2 lg:tw-px-0">
+			<img class="tw-w-4 lg:tw-w-6" src="@/assets/images/mfi_recommendations/earth.svg" alt="">
```

Comment:
I notice some blank `alt` tags in this template. We might want to fill these.

Reply:
They were filled, thanks!